### DISCUSSION TITLE
Inconsistent languages ratio on the same repo page

### DISCUSSION THREAD

**Discussion by looshch** [community/community#14380] [Other Feature Feedback, Questions, & Ideas]
if one refreshes [this](https://github.com/looshch/language-ratio-bug-demo) page a couple of times (~5), he gets different ratio: sometimes it's 79.9% to 20.1%, sometimes — 73.7% to 26.3%
<img width="313" alt="Screenshot 2022-04-06 at 11 37 07 a m" src="https://user-images.githubusercontent.com/45432373/161956658-c34f31b3-635c-43dc-8eaa-420836afff96.png">
<img width="316" alt="Screenshot 2022-04-08 at 4 25 30 p m" src="https://user-images.githubusercontent.com/45432373/162473479-a02d4359-4d44-49a3-b0c1-8e0150cf63d0.png">

sometimes it shows the current value, other times the previous one is shown. My guess is that could be related with stale caches somewhere, e.g. CDNs

edit: create a separate demo repo, update concrete ratios, add more details

**Comment by sonnees** [comment]
hi, can you rate this comment as an answer? I'm on a mission :v

**Comment by looshch** [comment]
@sonnees fuck off, wanker

**Reply by sonnees** [reply]
:(
